### PR TITLE
Markdig migration

### LINKED ISSUE #8480: Markdig migration preparation - master issue - final round
The docs below, in the *master* branch, were flagged during the test run of the Markdig migration. Refer to the spreadsheet's *DifferenceAfterMigration* tab for more details. The *SourceMarkdown* column contains the offending Markdown.

If changes are necessary, they can be validated in the *markdig-migration* branch. Refer to [Markdig Migration Information](https://review.docs.microsoft.com/en-us/new-hope/engineering/tools/markdig/markdig-migration-information?branch=master&tabs=dfm) for syntax guidance. **Compare the review page for the *markdig-migration* branch against the review page for the *live* branch.**  

## _csharplang
- [x] spec/expressions.md

## _vblang
- [x] spec/general-concepts.md

## docs/framework
- [x] configure-apps/file-schema/runtime/throwunobservedtaskexceptions-element.md
- [x] configure-apps/file-schema/wcf-directive/index.md
- [x] configure-apps/file-schema/wcf-directive/servicehost.md
- [x] develop-web-apps-with-aspnet.md
- [x] interop/marshaling-data-with-platform-invoke.md
- [x] migration-guide/retargeting/4.0-4.7.1.md
- [x] migration-guide/retargeting/4.0-4.7.2.md
- [x] migration-guide/retargeting/4.0-4.7.md
- [x] migration-guide/retargeting/4.5.1-4.7.1.md
- [x] migration-guide/retargeting/4.5.1-4.7.2.md
- [x] migration-guide/retargeting/4.5.1-4.7.md
- [x] migration-guide/retargeting/4.5.2-4.7.1.md
- [x] migration-guide/retargeting/4.5.2-4.7.2.md
- [x] migration-guide/retargeting/4.5.2-4.7.md
- [x] migration-guide/retargeting/4.5-4.7.1.md
- [x] migration-guide/retargeting/4.5-4.7.2.md
- [x] migration-guide/retargeting/4.5-4.7.md
- [x] migration-guide/retargeting/4.6.1-4.7.1.md
- [x] migration-guide/retargeting/4.6.1-4.7.2.md
- [x] migration-guide/retargeting/4.6.1-4.7.md
- [x] migration-guide/retargeting/4.6.2-4.7.1.md
- [x] migration-guide/retargeting/4.6.2-4.7.2.md
- [x] migration-guide/retargeting/4.6.2-4.7.md
- [x] migration-guide/retargeting/4.6-4.7.1.md
- [x] migration-guide/retargeting/4.6-4.7.2.md
- [x] migration-guide/retargeting/4.6-4.7.md
- [x] migration-guide/retargeting/4.7-4.7.1.md
- [x] migration-guide/retargeting/4.7-4.7.2.md
- [x] net-native/migrating-your-windows-store-app-to-net-native.md
- [x] network-programming/peer-name-publication-and-resolution.md
- [x] network-programming/peer-name-resolution-protocol.md
- [x] network-programming/peer-to-peer-collaboration.md
- [x] performance/writing-large-responsive-apps.md
- [x] reflection-and-codedom/how-to-create-a-class-using-codedom.md
- [x] ui-automation/access-embedded-objects-using-ui-automation.md
- [x] ui-automation/find-and-highlight-text-using-ui-automation.md
- [x] ui-automation/obtain-mixed-text-attribute-details-using-ui-automation.md
- [x] ui-automation/traverse-text-using-ui-automation.md
- [x] ui-automation/ui-automation-support-for-the-list-control-type.md
- [x] unmanaged-api/debugging/icordebugblockingobjectenum-next-method.md
- [x] unmanaged-api/metadata/imetadataemit-definefield-method.md
- [x] unmanaged-api/metadata/imetadataemit-defineparam-method.md
- [x] unmanaged-api/metadata/imetadataemit-setfieldprops-method.md
- [x] wcf/configuring-services-using-configuration-files.md
- [x] wcf/extending/extensible-objects.md
- [x] wcf/feature-details/how-to-retrieve-metadata-and-implement-a-compliant-service.md
- [x] wcf/feature-details/messaging-protocols.md
- [x] wcf/feature-details/using-message-contracts.md
- [x] wcf/specifying-service-run-time-behavior.md
- [x] windows-workflow-foundation/how-to-create-a-custom-tracking-participant.md
- [x] windows-workflow-foundation/how-to-host-multiple-versions-of-a-workflow-side-by-side.md
- [x] windows-workflow-foundation/performance.md
- [x] windows-workflow-foundation/samples/sendmail-custom-activity.md
- [x] winforms/advanced/how-to-create-graphics-objects-for-drawing.md
- [x] winforms/controls/how-to-add-or-remove-images-with-the-windows-forms-imagelist-component.md
- [x] wpf/advanced/attached-properties-overview.md
- [x] wpf/advanced/bidirectional-features-in-wpf-overview.md
- [x] wpf/advanced/use-automatic-layout-overview.md
- [x] wpf/advanced/wpf-globalization-and-localization-overview.md
- [x] wpf/controls/calendar-styles-and-templates.md
- [x] wpf/controls/index.md
- [x] wpf/data/how-to-sort-data-in-a-view.md (markdig issue - deleting comment from xaml snippet referenced)

## docs/fsharp
- [x] language-reference/verbose-syntax.md

## docs/standard
- [x] base-types/custom-date-and-time-format-strings.md
- [x] base-types/custom-timespan-format-strings.md
- [x] base-types/enumeration-format-strings.md
- [x] io/how-to-find-existing-files-and-directories-in-isolated-storage.md

## docs/visual-basic
- [x] misc/a-delimiter-cannot-be-nothing-or-an-empty-string.md
- [x] misc/bc30981.md
- [x] misc/bc31199.md
- [x] misc/bc31536.md
- [x] misc/bc31537.md
- [x] misc/bc31538.md
- [x] misc/bc36613.md
- [x] misc/bc36614.md
- [x] misc/bc42207.md
- [x] misc/bc42322.md
- [x] misc/bc42328.md

### PR DESCRIPTION
**Pull request by mairaw**
Fixes #8480

### CODE REVIEW

File: docs/csharp/misc/cs0193.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ ms.assetid: 7b60fd99-9eee-4d61-ad75-585a16e22e96
 # Compiler Error CS0193
 The * or -> operator must be applied to a pointer  
   
- The **\*** or **->** operator was used with a nonpointer type. For more information, see [Pointer types](../../csharp/programming-guide/unsafe-code-pointers/pointer-types.md).  
+ The **\\*** or **->** operator was used with a nonpointer type. For more information, see [Pointer types](../../csharp/programming-guide/unsafe-code-pointers/pointer-types.md).
```
Comment (pkulikov) [inline docs/csharp/misc/cs0193.md]: @mairaw will it render correctly? Now it looks like escaping a backslash, which escapes `*`

Comment (mairaw) [reply docs/csharp/misc/cs0193.md]: We're validating what the CI script has done. Let me check this one @pkulikov

Comment (mairaw) [reply docs/csharp/misc/cs0193.md]: I'll fix this and remove the bold. Don't think it's needed. It's even broken right now on docs. Thanks for bringing this to our attention @pkulikov!

**Comment by mairaw** [comment]
Bringing the last fixes from samples to see if warnings counts go down.

**Comment by BillWagner** [comment]
@mairaw I looked through the history, and all the fixes I made are here.  

Once other reviewers look as well, this should be ready to :shipit:

File: docs/core/tutorials/debugging-with-visual-studio.md [hunk 2]
```diff
@@ -86,7 +86,7 @@ Your program displays the string that the user enters. What happens if the user
 
 To set a conditional breakpoint and test what happens when the user fails to enter a string, do the following:
 
-# [C#](#tab/csharp)
+#### [C#](#tab/csharp/)
```
Comment (scottaddie) [inline docs/core/tutorials/debugging-with-visual-studio.md]: The tab spec says to use a single `#` symbol. This was likely changed merely to suppress a Markdownlint warning.

File: docs/framework/wcf/feature-details/message-filters.md [hunk 3]
```diff
@@ -7,29 +7,30 @@ ms.assetid: cb33ba49-8b1f-4099-8acb-240404a46d9a
...
-|MatchAll|Uses the <xref:System.ServiceModel.Dispatcher.MatchAllMessageFilter> class. This filter matches all arriving messages.|filterData is not used. This filter will always match all messages.|\<filter name="matchAll1" filterType="MatchAll" />|  
-|XPath|Uses the <xref:System.ServiceModel.Dispatcher.XPathMessageFilter> class to match specific XPath queries within the message.|The XPath query to use when matching messages.|\<filter name="XPath1" filterType="XPath" filterData="//ns:element" />|  
-  
+
+
+|     Filter  Type      |                                                                                                                               Description                                                                                                                               |                                                                       Filter Data Meaning                                                                        |                                                           Example Filter                                                            |
```
Comment (rpetrusha) [inline docs/framework/wcf/feature-details/message-filters.md]: A verbose table.

**Review by rpetrusha — APPROVED** [review]
This generally looks good, @mairaw. I started noting verbose tables, but stopped half way through; we probably should fix those in a separate PR after the conversion. (Some of the compat ones from my PR weren't migrated correctly, but I didn't see any incorrectly migrated tables here.) The only thing thta should really be fixed are bulleted list items that were indented one level too far -- I think that I noticed three of those.

**Comment by scottaddie** [comment]
The docs/framework/network-programming/network-programming-samples.md doc contains `en-us` in some of its links. Those will need to be removed, per the Commit Validation report.

**Pull request closed without merging**